src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -78,7 +78,8 @@
     public static final String WINRM_SPN_USE_HTTP = "winrm-spn-use-http";
     public static final String WINRM_LOCALE = "winrm-locale";
     public static final String WINRM_TIMEOUT = "winrm-timeout";
-    
```

Comment:
(this difference is odd; double check the line endings of the file and the ones you are inserting. Isn't the file using \r\n, and you inserting only \n, or the inverse?)

Reply:
It is a cosmetics change: spaces on empty line

Reply:
no cosmetics change in a purposeful change; please remove